CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Change Log
 
+This release adds support for the following new devices:
+
+* Xiaomi Zero Fog Humidifier \(shuii.humidifier.jsq001\)
```

Comment:
Please remove this for now. The changelog (or parts of it) will be automatically generated as part of the release procedures, which is the time when the high-level changelog gets also updated.

Reply:
Done